ajax/libs/adapter/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "adapter",
+  "keywords": [
```

Comment:
1. Do you create these keywords yourself? I think there are some official keywords: https://github.com/webrtc/adapter/blob/v3.0.0/bower.json#L19-L30

2. Let's put `keywords` a little bit lower place in `package.json`, reserve the upper place for those most important info.